tests/unit_tests/udf_wrapper_params/sequence_classification/multiple_model_multiple_batch_complete.py

```diff
@@ -47,18 +89,34 @@ class MultipleModelMultipleBatchComplete:
             logits=logits)
     })
 
-    inputs = [("sub_dir1", "model1", "My test text")] * data_size + \
-             [("sub_dir2", "model2", "My test text")] * data_size
-
-    outputs = [("sub_dir1", "model1", "My test text", "label1", 0.21),
-               ("sub_dir1", "model1", "My test text", "label2", 0.24),
-               ("sub_dir1", "model1", "My test text", "label3", 0.26),
-               ("sub_dir1", "model1", "My test text", "label4", 0.29)
-               ] * data_size + \
-              [("sub_dir2", "model2", "My test text", "label1", 0.21),
-               ("sub_dir2", "model2", "My test text", "label2", 0.24),
-               ("sub_dir2", "model2", "My test text", "label3", 0.26),
-               ("sub_dir2", "model2", "My test text", "label4", 0.29)
-               ] * data_size
-
-    udf_wrapper = udf_wrapper
+    inputs_single_text = \
+        [("sub_dir1", "model1", "My test text")] * data_size + \
```

Comment:
I suggest to split the test definition, because we will get similar definitions for the other UDFs and then we should handle all of them the same. We can't put them all in the same file, so we put every in its own file, but you can use test modules to keep the names shorter.

Reply:
There will be a separate test file for each task. I already set the directories accordingly (../udf_wrappers_params/sequence_classification/).

Text pair and single text are in the same place as they are subtasks of sequence classification task. Others will have separate files.

Reply:
ok, that make sense